src/android/MyConnectionService.java

```diff
@@ -171,6 +173,9 @@ public void showWebApp(String userAction, String payload) {
     }
 
     public static Connection getConnection() {
+        if (activeOutgoingConnection != null) {
```

Comment:
need some code comment.

does this mean there should be ONLY one outgoing but you can have lots of incoming?

wouldn't this prevent accessing any of the incoming connections when there is an outgoing?

i think you mentioned elsewhere that the service could have multiple connections active at the same time?

Reply:
I believe (in general) you could have multiple connections, wanted to store both IncomingConnections and OutgoingConnections via the connectionMap (organized by call UUID) to support situations like this.

Currently we don't have a callUUID for the outgoing call (we could pass one from facetalk through sendCall)

I'll add a comment above explaining.

Reply:
Added a comment about the reasoning for activeOutgoingConnection.